### PR TITLE
facs: select analysis and condition

### LINKED ISSUE #163: NSF Exercise 2 - Flow cytometry setup page
The setup table needs to allow each sample to be analyzed by more than one condition (like the microscopy experimental technique, when applicable). This will correspond to more tabs in the small tabbed window for each sample once the user gets to the flow cytometry run/analyze page. Once a condition is selected, another 'select condition' drop down menu will appear below it in the same row (within the black horizontal lines). Subrows for a sample are divided by a light gray line. Like in the 7.06 exercise:
https://starcellbio.mit.edu/#view=microscopy&experiment_id=scb_Experiment_mf96aa1ogjgj3&assignment_id=assignment_706_2014_ps2&microscopy_id=scb_Microscopy_1896mm1c52jfp

### PR DESCRIPTION
**Pull request by annagav**
Fix #163.

### CODE REVIEW

File: html_app/ui/facs.gss [hunk 1]
```diff
@@ -381,14 +381,9 @@ background-color:#676767 !important;
 }
 
 .scb_s_facs_cell_treatment_radio{
```
Comment (pdpinch) [inline html_app/ui/facs.gss]: Do we still need this CSS class?

File: html_app/ui/facs.gss [hunk 2]
```diff
@@ -381,14 +381,9 @@ background-color:#676767 !important;
...
-	position: absolute;
-	top: 4px;
 }
 
 .scb_s_facs_cell_treatment_text{
```
Comment (pdpinch) [inline html_app/ui/facs.gss]: Same here -- if the class is empty, do we need it?

**Comment by pdpinch** [comment]
This is working fine for me.

**Comment by pdpinch** [comment]
@annagav Just the couple of questions.

**Comment by annagav** [comment]
Updated!

**Pull request merged**